src/main/java/temporal/OrderProcessingActivityImpl.java

```diff
@@ -0,0 +1,20 @@
+package temporal;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import temporal.model.OrderInfo;
+
+public class OrderProcessingActivityImpl implements OrderProcessingActivity {
+
+  private static final Logger logger = LoggerFactory.getLogger(OrderProcessingActivityImpl.class);
+
+  @Override
+  public void add(OrderInfo orderInfo) {
+    logger.info("Order added in Activity: " + orderInfo.toString());
```

Comment:
Rather than string concatenation, it is recommended to use the string parameterization format for log statements

Reply:
@reselbob You do not need to use String.format explicitly. slf4j loggers will substitute values for the {} placeholders in the parameterized string template

Reply:
addressed